cypress/integration/institution.spec.js

```diff
@@ -15,18 +15,19 @@ describe('User adds an institution to their profile', () => {
 
```

Comment:
Det er merge conflicts her som må ryddes opp i

Reply:
Det går heller ikke an å GET'e fra cristininstitutions (men mest sannsynlig ikke frontend sin feil):
`{"error":"Parameter 'name' may only contain alphanumeric characters, dash and whitespace"}`

Reply:
eller blir det gjort et kall ved en feil her? dette er det kallet som feiler:
`https://api.dev.nva.aws.unit.no/cristin-institutions?name=194.0.0.0`

Reply:
og stemmer det at toppnivå ikke skal vises i visningen? har valgt NTNU på alle de under her, men det vises ikke.
<img width="801" alt="Skjermbilde 2020-02-11 kl  17 53 01" src="https://user-images.githubusercontent.com/6595982/74258943-672f2f00-4cf7-11ea-906f-9511f1bb62ef.png">

Reply:
Hmm, klarer ikke å gjenskape dette. Kallet https://api.dev.nva.aws.unit.no/cristin-institutions?name=194.0.0.0 er ivhertfall feil

Reply:
Da har jeg konferert med Nikolai og han fikser tjenesten slik at vi får de dataene vi trenger. Får med parent til enheten og da blir det lettere å få ut det vi trenger.